src/i-shared-sea-mapping.ts

```diff
@@ -0,0 +1,388 @@
+
+// minter-level events
+export function handleMinAuctionDurationSecondsUpdated(
+  event: MinAuctionDurationSecondsUpdated
+): void {
+  // load minter
+  const minter = loadOrCreateMinter(event.address, event.block.timestamp);
+
+  // update minter extra minter details value
+  setMinterExtraMinterDetailsValue(
+    "minAuctionDurationSeconds",
+    event.params.minAuctionDurationSeconds,
+    minter
+  );
+
+  // update minter's updatedAt timestamp to induce a sync, and save
+  minter.updatedAt = event.block.timestamp;
+  minter.save();
+}
+
+export function handleMinterTimeBufferUpdated(
+  event: MinterTimeBufferUpdated
+): void {
+  // load minter
+  const minter = loadOrCreateMinter(event.address, event.block.timestamp);
+
+  // update minter extra minter details value
+  setMinterExtraMinterDetailsValue(
+    "minterTimeBufferSeconds",
+    event.params.minterTimeBufferSeconds,
+    minter
+  );
+
+  // update minter's updatedAt timestamp to induce a sync, and save
+  minter.updatedAt = event.block.timestamp;
+  minter.save();
+}
+
+export function handleMinterRefundGasLimitUpdated(
+  event: MinterRefundGasLimitUpdated
+): void {
+  // load minter
+  const minter = loadOrCreateMinter(event.address, event.block.timestamp);
+
+  // update minter extra minter details value
+  setMinterExtraMinterDetailsValue(
+    "refundGasLimit",
+    BigInt.fromI32(event.params.refundGasLimit),
+    minter
+  );
+
+  // update minter's updatedAt timestamp to induce a sync, and save
+  minter.updatedAt = event.block.timestamp;
+  minter.save();
+}
+
+// project-level events
+export function handleConfiguredFutureAuctions(
+  event: ConfiguredFutureAuctions
+): void {
+  const minterProjectAndConfig = loadOrCreateMinterProjectAndConfigIfProject(
+    event.address, // minter
+    event.params.coreContract,
+    event.params.projectId,
+    event.block.timestamp
+  );
+  if (!minterProjectAndConfig) {
+    // project wasn't found, warning already logged in helper function
+    return;
+  }
+
+  const projectMinterConfig = minterProjectAndConfig.projectMinterConfiguration;
+  // update project minter configuration fields
+  projectMinterConfig.priceIsConfigured = true;
+  projectMinterConfig.basePrice = event.params.basePrice;
+  // update project minter configuration extraMinterDetails json field
+  setProjectMinterConfigExtraMinterDetailsValue(
+    "startTime",
+    event.params.timestampStart,
+    projectMinterConfig
+  );
+  setProjectMinterConfigExtraMinterDetailsValue(
+    "projectAuctionDurationSeconds",
+    event.params.auctionDurationSeconds,
+    projectMinterConfig
+  );
+  setProjectMinterConfigExtraMinterDetailsValue(
+    "minBidIncrementPercentage",
+    BigInt.fromI32(event.params.minBidIncrementPercentage),
+    projectMinterConfig
+  );
+  projectMinterConfig.save();
+
+  // induce sync if the project minter configuration is the active one
+  updateProjectIfMinterConfigIsActive(
```

Comment:
Definitely not something we need to change now, but we decided we want to start treating ProjectMinterConfigs as first class sync entities right?

Reply:
yup! There is an Asana polish task for that, and it specifically calls out some extra helpful benefits on the SEA minter 🙏  https://app.asana.com/0/1203634050572702/1205060294151483/f